tools/expat/Makefile

```diff
@@ -15,19 +15,19 @@ PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_HASH:=a247a7f6bbb21cf2ca81ea4cbb916bfb9717ca523631675f99b3d4a5678dcd16
 PKG_SOURCE_URL:=https://github.com/libexpat/libexpat/releases/download/R_$(subst .,_,$(PKG_VERSION))
 
+HOST_BUILD_PARALLEL:=1
+
 include $(INCLUDE_DIR)/host-build.mk
-include $(INCLUDE_DIR)/cmake.mk
-
-CMAKE_HOST_OPTIONS += \
-	-DDOCBOOK_TO_MAN=OFF \
-	-DEXPAT_BUILD_TOOLS=OFF \
-	-DEXPAT_BUILD_EXAMPLES=OFF \
-	-DEXPAT_BUILD_TESTS=OFF \
-	-DEXPAT_BUILD_DOCS=OFF \
-	-DEXPAT_WITH_LIBBSD=OFF \
-	-DEXPAT_ENABLE_INSTALL=ON \
-	-DEXPAT_DTD=ON \
-	-DEXPAT_NS=OFF \
-	-DEXPAT_DEV_URANDOM=OFF
+
+HOSTCC := $(HOSTCC_NOCACHE)
+
+HOST_CONFIGURE_ARGS += \
+	--disable-shared \
+	--without-docbook \
+	--with-pic
+
+define Host/Install
+	$(MAKE) -C $(HOST_BUILD_DIR) install
+endef
```

Comment:
this can be removed i think?

Reply:
Beats me. Was there originally.

Reply:
yeah it is outdated

default value of Host/Install expands to

```
	+$(HOST_MAKE_VARS) \
	$(MAKE) $(HOST_JOBS) -C $(HOST_BUILD_DIR) \
		$(HOST_MAKE_FLAGS) \
		install
```